spec/mailers/project_mailer_spec.rb

```diff
@@ -20,8 +20,8 @@
   end
 
   describe '#comment' do
-    let!(:comment) { create :comment, project: project }
-    let!(:commenter) { create(:comment, project: project).user }
+    let!(:comment) { create :comment, commentable: project }
+    let!(:commenter) { create(:comment, commentable: project).user }
```

Comment:
Totally! I was just changing the factories to get the tests pass, didn't look at the semantics. Will change it!